movies/presentation/src/main/java/com/example/movies/presentation/ui/movieDetail/MovieDetailFragment.kt

```diff
@@ -80,6 +84,8 @@ class MovieDetailFragment : BaseFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        //TODO implement viewModel call using SafeArg Argument passed through NavigationController
```

Comment:
I think this pending implementation should be completed as part of the movie list feature